ruff.toml

```diff
@@ -56,7 +56,7 @@ select = [
     "PLR0915", # keeping functions and methods short and focused
 ]
 
-[per-file-ignores]
+[lint.per-file-ignores]
```

Comment:
why do you need to add `lint` prefix?

Reply:
Because of a deprecation warning, I moved to the new prefix:
```
The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `ruff.toml`:
  - 'mccabe' -> 'lint.mccabe'
  - 'pylint' -> 'lint.pylint'
  - 'per-file-ignores' -> 'lint.per-file-ignores'
  ```